pvl/decoder.py

```diff
@@ -109,6 +137,14 @@ def raise_unexpected(self, stream, token=None):
     def raise_unexpected_eof(self, stream):
         self.raise_error('Unexpected EOF', stream)
 
+    def broken_parameter_value(self, lineno):
```

Comment:
Is "parameter_value" the best name for the thing that is broken?  Does the PVL spec use the words "parameter value"?  It would be best to stick to the language in the spec if possible